src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v2/FlowControllerV2.java

```diff
@@ -160,6 +162,50 @@
         return flowService.patchFlow(flowId, flowPatchDto);
     }
 
+    /**
+     * Get existing flow loops.
+     *
+     * @param flowId filter by flow id
+     * @param switchId filter by switch id
+     * @return list of flow loops
+     */
+    @ApiOperation(value = "Get flow loops", response = FlowLoopResponse.class, responseContainer = "List")
+    @GetMapping(value = "/loop")
+    @ResponseStatus(HttpStatus.OK)
+    public CompletableFuture<List<FlowLoopResponse>> getFlowLoops(
```

Comment:
Why do you use the plural form (`getFlowLoopS`)? A flow can't have more than one loop
Probably would be better to use the singular form (`getFlowLoopS -> getFlowLoop`)

Reply:
This method allows you to get all looped flows in the system by combining filters. You can skip optional flow_id param and get looped flows by switch_id or all of them.